### ISSUE TITLE
CMST 2021.12.02 on AUR: error making target '../../translations/cmst_pl_PL.qm', needed by 'qrc_cmst.cpp'

### ISSUE THREAD

**Issue by DanielPrjct** [andrew-bibb/cmst#258]
Dear Andrew:

I'm the current maintainer of CMST on AUR, because it was orphaned some months ago.
I've wrote the PKGBUILD for the new 2021.12.02 release, and testing it in my Arch Linux system.
Apparently, there isan error on the new CMST code: makepkg -sri downloads the source code and beggins to compile, but aborts with following messages:
make[1]: *** No rule to make target '../../translations/cmst_pl_PL.qm', needed by 'qrc_cmst.cpp'.  Stop.
make[1]: Leaving directory '/home/daniel/Builds/cmst/src/cmst-2021.12.02/apps/cmstapp'
make: *** [Makefile:48: sub-apps-cmstapp-make_first] Error 2
I have no knowledge on how to fix this issue. Could you check your code, please?
Attached, my PKGBUILD:

---
# Maintainer: Alad Wenter <alad@archlinux.org>
# Contributor: Andrew Bibb <ajbibb@myfairpoint.net>
# Contributor: DanielH <danielmirkin at gmail dot com>

pkgname=cmst
pkgver=2021.12.02
pkgrel=1
pkgdesc="A QT based GUI front end for the connman connection manager"
arch=('x86_64')
url="https://github.com/andrew-bibb/cmst"
license=('custom:MIT')
depends=('qt5-base' 'connman' 'libxkbcommon-x11' 'hicolor-icon-theme')
source=("$url/releases/download/$pkgname-$pkgver/$pkgname-$pkgver.tar.xz")
sha256sums=('c799145e699db227c8af0feb9db38bc31ce1423173b84e549205a273db8b98b5')

prepare() {
	cd "$pkgname-$pkgver"
	sed -i '/^conf.path =/c conf.path = /usr/share/dbus-1/system.d' apps/rootapp/rootapp.pro
}

build() {
	cd "$pkgname-$pkgver"
	qmake-qt5 DISTRO=arch
	make 
}

package() {
	cd "$pkgname-$pkgver"
	make INSTALL_ROOT="$pkgdir/" install
}

---
Your response will be welcome!
Best regards, DanielH

**Comment by andrew-bibb** [comment]
Not a surprise. I was doing a lot of work on the translation building code in qmake past week or so.  I wanted qmake to run lrelease automatically and finally figured it out.  Doing it was supposed to be simple, and it was once the handful of commands were put in the proper locations.  It was finding the proper location that took an embarrassingly long time.

I just now, as in 2 or 3 minutes ago, uploaded version  2021.12.18-2 of the program.  The automatic build of translations should work (actually been fixed for the past few days) and I also just implemented the feature in issue #235.  This version should be working.  Note that while the readme says to run "qmake", "lrelease", "make" then "make install" that is incorrect for this version.  I'm leaving it there for now because it is correct for the last released version, but if downloading 2021.12.18 from Github skip the "lrelease" step.  It is done automatically now.

**Comment by andrew-bibb** [comment]
One further point.  I've moved a bunch of things around so it would be best to download and build this in a clean directory.  In particular the file cmst.qrc should not be an old version.  It is fixed now here on Github but that is the file that used to contain reference that is causing the error to be thrown.  

All of the enumerations of translation (*.qm) files have now been removed from the manual resource file (cmst.qrc) and a resource file containing them is generated automatically during the build.

**Comment by andrew-bibb** [comment]
Just published the 2022.01.05 release.  I also successfully built the release using this PKGBUILD modified from the one you have in the AUR:

```
# Maintainer: Alad Wenter <alad@archlinux.org>
# Contributor: Andrew Bibb <ajbibb@outlook.com>
pkgname=cmst
pkgver=2022.01.05
pkgrel=1
pkgdesc="A QT based GUI front end for the connman connection manager"
arch=('x86_64')
url="https://github.com/andrew-bibb/cmst"
license=('custom:MIT')
depends=('qt5-base' 'connman' 'libxkbcommon-x11' 'hicolor-icon-theme')
source=("$url/releases/download/$pkgname-$pkgver/$pkgname-$pkgver.tar.xz")
sha256sums=('6ddc0e3a702592ec98b80164aa2b5ad1edb66d3f8dce18a545f26a67a46dc13a')
prepare() {
	cd "$pkgname-$pkgver"
	sed -i '/^conf.path =/c conf.path = /usr/share/dbus-1/system.d' apps/rootapp/rootapp.pro
}

build() {
	cd "$pkgname-$pkgver"
	qmake-qt5 DISTRO=arch
	make
}


package() {
	cd "$pkgname-$pkgver"
	make INSTALL_ROOT="$pkgdir/" install
}
```


I made a few changes (my email address, sha256sum, pkgver, and file path) to the one you had in the AUR. Downloaded and built fine here.  Might want to change the maintainer name at the top as I don't think that is you.

Should also verify the depends list is current.  The QT package keeps getting broken up into pieces in Arch and I am not sure that -base now includes everything needed.  It may, but I'm not sure.
